ProxyServer/k6ProxyScript.js

```diff
@@ -0,0 +1,8 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
+
+export default function() {
+  http.get(`http://localhost:5001?4913406`);
```

Comment:
Did you not need a '/' in between '5000' and '?'?

Reply:
Nice catch!! that may be why my proxy was faster. I'm going to change and retest